ftw/contentpage/tests/pages.py

```diff
@@ -1,45 +1,48 @@
-from ftw.testing import browser
-from ftw.testing.pages import Plone
+from plone import api
 
 INDEX_XPATH = '//*[contains(concat(" ", normalize-space(@class), " "), ' + \
     '" letter-index ")]'
 
 
-class SubjectListingView(Plone):
+class SubjectListingView(object):
+    
+    def __init__(self, browser):
+        self.browser = browser
+        self.portal = api.portal.get()
 
     def visit(self):
-        self.visit_portal('@@subject-listing')
-        self.assert_body_class('template-subject-listing')
+        self.browser.visit(self.portal, view='@@subject-listing')
+        assert 'template-subject-listing' in self.browser.css('body').first.attrib['class']
```

Comment:
This matches `<div class="not-really-a-template-subject-listing-item">`.
Use http://ftwtestbrowser.readthedocs.io/en/1.23.1/api_nodes_forms.html#ftw.testbrowser.nodes.NodeWrapper.classes 😎 

Also use `self.assertIn` because this will have a useful assertion message.

```python
self.assertIn('x', browser.css('body').first.classes)
```